components/calculation/MeasureHighlightingPanel.tsx

```diff
@@ -12,20 +18,146 @@ const useStyles = createStyles({
+    return measureBundle.content?.entry?.find(e => e.resource?.resourceType === 'Measure')?.resource as fhir4.Measure;
+  }, [measureBundle]);
 
-  return (
-    <>
-      <div className={classes.highlightedMarkup}>{parse(measureReportLookup[patientId]?.text?.div || '')}</div>
-    </>
-  );
+  /**
+   * Creates an object where the population is the key and the value is 0 or 1:
+   * 0 if that population is not a desired population and 1 if it is.
+   */
+  function constructDesiredPopulationsValuesArray(measurePopulations: MultiSelectData[]) {
+    const desiredPopulations: DesiredPopulations = {};
+    measurePopulations.map(measurePopulation => {
+      if (currentPatients[patientId].desiredPopulations?.includes(measurePopulation.value)) {
+        desiredPopulations[measurePopulation.label as string] = 1;
+      } else {
+        desiredPopulations[measurePopulation.label as string] = 0;
+      }
+    });
+
+    return desiredPopulations;
+  }
+
+  /**
+   * Creates an object where the population is the key and it has two properties: actual and desired.
+   * Both properties can either be 0 or 1: 0 if that population is not desired (or actual) and 1 if
+   * that population is desired (or actual).
+   */
+  function constructBothPopulationsValuesArray(group: MeasureReportGroup | undefined) {
+    const desiredPopulations = constructDesiredPopulationsValuesArray(getMeasurePopulations(measure));
+    const bothPopulations: BothPopulations = {};
+    group?.population?.map(population => {
+      const key = population?.code?.coding?.[0]?.display || population?.code?.coding?.[0]?.code;
+      if (key) {
+        bothPopulations[key as string] = { desired: desiredPopulations[key as string], actual: population?.count };
+      }
+    });
+    return bothPopulations;
+  }
+
+  if (measure) {
+    const group = measureReportLookup[patientId]?.group?.[0];
+    const bothPopulations = constructBothPopulationsValuesArray(group);
+
+    return (
+      <>
+        <div />
+        <Group>
+          <div style={{ paddingRight: 5 }}>
+            <Text size="xl" weight={700}>
+              Population Comparison Table
+            </Text>
+          </div>
+          <div>
+            <Popover opened={opened} onClose={() => setOpened(false)} width={500}>
+              <Popover.Target>
+                <ActionIcon aria-label={'More Information'} onClick={() => setOpened(o => !o)}>
+                  <InfoCircle size={20} />
+                </ActionIcon>
+              </Popover.Target>
+              <Popover.Dropdown>
+                The rows in this table are the populations and the columns two columns represent the desired and actual
+                populations. The contents are either 0 or 1: 0 if that population does not exist, 1 if it does. If any
+                of the rows are highlighted red, that means the desired population does not match the actual population.
+                If they are highlighted green, then they match.
```

Comment:
I can't suggest a multiline comment here, so I'll just add some edits in a regular comment :)

```
The rows in this table represent the possible populations as specified on the uploaded Measure and the two columns represent the desired and actual populations as stored/calculated by FQM-Testify. For the 'Actual' column, the value will be 1 if the patient calculates into the respective population, and 0 if it does not. For the 'Desired' column, the value will be 1 if the respective population was selected as a desired population for the patient, and 0 if not. If any of the rows are highlighted red, that means the desired population does not match the actual population.
If they are highlighted green, then they match.
```